src/child/components/Favourite.js

```diff
@@ -95,9 +106,11 @@ class Favourite extends Component {
               onDrop={e => bindings.dnd.onDrop(e, stockCode)}
               onDragEnd={this.onDragEnd}
               onDoubleClick={() => bindings.onDoubleClick(stockCode, name)}
+              onMouseOver={this.onMouseOver}
+              onMouseOut={this.onMouseOut}
             >
                 <div className="drop-target">
-                    <div className={`darkens favourite tearable ${cls}`} draggable="false">
+                    <div className={`darkens favourite tearable ${cls} ${isHovered ? 'hovered' : ''}`} draggable="false">
```

Comment:
I think it would be a bit cleaner and more consistent to make use of the `cls` const above and revert the change on this line.

```js
const cls = classNames({
    selected,
    dark: selected,
    hovered: isHovered
});
```